src/main/java/edu/greenblitz/robotName/utils/GBMath.java

```diff
@@ -15,6 +18,10 @@ public class GBMath {
 	public static double absoluteModulo(double x, double y) {
 		return ((x % y) + y) % y;
 	}
-	
-	
+
+	public static double reverseAngle(double angleInRadians){
```

Comment:
maybe if you would use the Rotation2D object for your calculations' you would have find out that it has a reverse function in it.